toolchain/check/import_ref.cpp

```diff
@@ -73,6 +102,16 @@ class ImportRefResolver {
   }
 
  private:
+  // A step in work_stack_.
+  struct Work {
+    // The instruction to work on.
+    SemIR::InstId inst_id;
+
+    // True if an incomplete type has been generated; only used by instructions
+    // with incomplete types.
+    bool incomplete_type = false;
```

Comment:
What do you think about calling this `resolved_incomplete_type` to make it a bit more clear how it is used? See also the naming suggestion below.

Also happy with other names if a different approach would work.

Reply:
Renamed to `made_incomplete_type`, hoping it helps (`resolved` just implies to me a more complete state)